packages/verification-vue-component/src/components/Verification/VerificationItem.vue

```diff
@@ -3,7 +3,12 @@
          :class="{ 'confirmation-step': digitalTwin.confirmationStep, 'error': digitalTwinInformation.error }">
         <div class="card-container">
             <div v-if="digitalTwinInformation.active" class="header">
-                <img src="../../assets/img/certifaction_logo.svg" alt="Certifaction"/>
+                <template v-if="urlTypeLoaded === 'HIN'">
+                    <img src="../../assets/img/hin-logo.svg" alt="Hin"/>
+                </template>
+                <template v-else>
+                    <img src="../../assets/img/certifaction_logo.svg" alt="Certifaction"/>
+                </template>
```

Comment:
This package is public which can be downloaded and used by anyone. I don't think we should have customer specific (HIN) code in here. ATM the logo is the only thing (beside the styles) which is custom, so I probably would pass it as prop to this package or maybe it's even possible to have it as a css variable (background image) as well. Then no additional prop would be needed and we have this clean only css variable solution